src/main/java/org/folio/rest/impl/OaiPmhViewInstancesAPI.java

```diff
@@ -80,6 +94,28 @@ public void getOaiPmhViewInstances(String startDate, String endDate, boolean del
     }
   }
 
+  @Override
+  public void postOaiPmhViewEnrichedInstances(OaipmhInstanceIds entity, RoutingContext routingContext,
```

Comment:
`@Validate` is missing. The .raml file says `is: [validate]`.

Reply:
It looks genetared by raml OaiPmhView inteface has this method annotated with @Validate. Is it not sufficient to allow implementation to have this annotation too?
![image](https://user-images.githubusercontent.com/63074477/88069990-d5b3cb00-cb7a-11ea-8780-2d8316d2875a.png)

Reply:
RMB only looks at the implementation, not the interface, to decide whether to do validation or not.